h2o-py/tests/testdir_munging/pyunit_6987_merge_one_empty.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+import pandas as pd
+
+def mergeOneEmptyFrame():
+    # PUBDEV-6987: merge with one empty frame and one normal frame.
+    file1 = h2o.H2OFrame(pd.DataFrame({"A1":[1], "A2":[0]}))
+    file3 = h2o.H2OFrame(pd.DataFrame({"A1":[1], "A2":[0]}))
+    file2 = file3.drop([0], axis=0) # create frame with empty content, just the headers
+
+    f1Mergef2 = file1.merge(file2)
+    f2Mergef1 = file2.merge(file1)
+    f2Mergef2 = file2.merge(file2)  # merging of empty frame with just headers
+
+    assert f1Mergef2[0,0]==file1[0,0] and f1Mergef2[0,1]==file1[0,1], "Merge file error!"
```

Comment:
this should be an empty frame I think

Reply:
It is an empty frame but do contain headers  That is what that assert statement is comparing.

Reply:
I still think that an explicit assert for nrows would add clarity to the test